seedpod_ground_risk/layers/osm_tag_layer.py

```diff
@@ -3,8 +3,12 @@
 
 import geopandas as gpd
 import numpy as np
+import requests
 import shapely.geometry as sg
 from holoviews.element import Geometry
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
Add a TODO and a comment here explaining why for down the line when we inevitably forget